### DISCUSSION TITLE
What units to use for the discharge of pumping wells?

### DISCUSSION THREAD

**Discussion by mbakker7** [pastas/pastas#366] [Q&A] [answered]
The units of pumping wells can vary over fast orders of magnitude. For some time series, the discharge of an entire well field is summed, while for others wells, it may simply be a single irrigation well. How does Pastas pick appropriate initial values for the parameters of the response function? Are there preferred units for well discharges?

**Comment by dbrakenhoff** [comment, marked as answer]
I often convert the units of my pumping well timeseries to Mm^3/year, but that is mostly for relatively large extractions, and yields somewhat comprehensible (as in, not very very small) estimates of the gain. I guess a general suggestion could be to pick units so that your gain is somewhere in the order of cm or m per unit pumping?

The only initial parameter that is adjusted according to the stress is the `A` and is estimated by 1/meanstress (where meanstress = std(stress)). The other parameters always use the same default initial values.
